packages/hoprd/src/index.ts

```diff
@@ -86,7 +86,8 @@ function generateNodeOptions(argv: CliArgs, environment: ResolvedEnvironment): H
     options.strategy = StrategyFactory.getStrategy(argv.default_strategy)
     if (argv.max_auto_channels !== undefined) {
       options.strategy.configure({
-        max_channels: argv.max_auto_channels
+        max_channels: argv.max_auto_channels,
+        auto_redeem_tickets: argv.auto_redeem_tickets
```

Comment:
So `auto_redeem_tickets` is only checked if `max_channels` is set to a custom value, and thus deviates from default *sqrt(N)* setting

Reply:
Good catch, missed the condition so will improve it.